client/my-sites/checkout/composite-checkout/payment-methods/id-wallet.js

```diff
@@ -0,0 +1,367 @@
+				case 'FIELD_VALUE_SET':
+					return {
+						...state,
+						fields: {
+							...state.fields,
+							[ action.payload.key ]: {
+								value: maskField(
+									action.payload.key,
+									state.fields[ action.payload.key ],
+									action.payload.value
+								),
+								isTouched: true,
+								errors: [],
+							},
+						},
+					};
+				case 'FIELD_ERROR_SET':
+					return {
+						...state,
+						fields: {
+							...state.fields,
+							[ action.payload.key ]: {
+								...state.fields[ action.payload.key ],
+								errors: [ action.payload.message ],
+							},
+						},
+					};
+				case 'TOUCH_ALL_FIELDS':
+					return {
+						...state,
+						fields: Object.keys( state.fields ).reduce(
+							( obj, key ) => ( {
+								...obj,
+								[ key ]: {
+									...state.fields[ key ],
+									isTouched: true,
+								},
+							} ),
+							{}
+						),
+					};
+			}
+			return state;
+		},
+		actions,
+		selectors,
+	} );
+
+	return { ...store, actions, selectors };
+}
+
+export function createIdWalletMethod( { store, stripe, stripeConfiguration } ) {
+	return {
+		id: 'id-wallet',
+		label: <IdWalletLabel />,
+		activeContent: <IdWalletFields stripe={ stripe } stripeConfiguration={ stripeConfiguration } />,
+		submitButton: (
+			<IdWalletPayButton
+				store={ store }
+				stripe={ stripe }
+				stripeConfiguration={ stripeConfiguration }
+			/>
+		),
+		inactiveContent: <IdWalletSummary />,
+		getAriaLabel: () => 'OVO',
+	};
+}
+
+function IdWalletFields() {
+	const { __ } = useI18n();
+
+	const fields = useSelect( ( select ) => select( 'id-wallet' ).getFields() );
+	const getField = ( key ) => fields[ key ] || {};
+	const getFieldValue = ( key ) => getField( key ).value ?? '';
+	const getErrorMessagesForField = ( key ) => {
+		const managedValue = getField( key );
+		if ( managedValue?.isRequired && managedValue?.value === '' ) {
+			return [ __( 'This field is required.' ) ];
+		}
+		return managedValue.errors ?? [];
+	};
+	const { setFieldValue } = useDispatch( 'id-wallet' );
+
+	const customerName = useSelect( ( select ) => select( 'id-wallet' ).getCustomerName() );
+	const { changeCustomerName } = useDispatch( 'id-wallet' );
+	const { formStatus } = useFormStatus();
+	const isDisabled = formStatus !== 'ready';
+	const countriesList = useCountryList( [] );
+
+	return (
+		<IdWalletFormWrapper>
+			<IdWalletField
+				id="id-wallet-cardholder-name"
+				type="Text"
+				autoComplete="cc-name"
+				label={ __( 'Your name' ) }
+				value={ customerName?.value ?? '' }
+				onChange={ changeCustomerName }
+				isError={ customerName?.isTouched && customerName?.value.length === 0 }
+				errorMessage={ __( 'This field is required' ) }
+				disabled={ isDisabled }
+			/>
+			<div className="id-wallet__contact-fields">
+				<CountrySpecificPaymentFieldsUI
+					countryCode={ 'ID' } // If this payment method is available and the country is not India, we have other problems
```

Comment:
This string is correct, but the comment is not. Might as well remove the comment, I guess?